bin/raptor

```diff
@@ -47,6 +47,20 @@ var parent = parser
     default: '',
     transform: utils.fromEnvironment('ADB_PORT')
   })
+  .option('marionetteHost', {
+    full: 'marionette-host',
+    metavar: '<host>',
+    help: 'connect to marionette on a remote host. tip: use with --marionette-port. Envrionment: MARIONETTE_HOST',
+    default: 'localhost',
+    transform: utils.fromEnvironment('MARIONETTE_HOST')
+  })
+  .option('marionettePort', {
+    full: 'marionette-port',
+    metavar: '<port>',
+    help: 'set port for connecting to marionette on a remote host. tip: use with --marionette-host. Environment: MARIONETTE_PORT',
```

Comment:
Nit: don't need the word 'set', same for --adb-port, makes the help text a bit shorter